plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/context/lightweight-seqd.contexts

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<contexts:Context xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:constraints="http://www.eclipse.org/papyrus/constraints/0.9" xmlns:contexts="http://www.eclipse.org/papyrus/properties/contexts/0.9" xmi:id="_70QJkKDJEeSZxfCXzZz3-w" name="Lightweight Sequence Diagram" label="Sequence Diagram">
+  <tabs xmi:id="_70QJkaDJEeSZxfCXzZz3-w" label="Sequence Diagram" id="lw-seqd" category="org.eclipse.papyrus.seqd" image="" priority="8">
+    <!-- if it shouldn't replace the UML tab, don't use the same section names as in UML.ctx; I postfixed them with 1 -->
+    <sections xmi:id="_70QJkqDJEeSZxfCXzZz3-w" name="SingleMessage1" sectionFile="ui/SingleMessage.xwt">
+      <widget href="ui/SingleMessage.xwt#/"/>
+    </sections>
+    <sections xmi:id="_70QJk6DJEeSZxfCXzZz3-w" name="MultipleMessage1" sectionFile="ui/MultipleMessage.xwt">
+      <widget href="ui/MultipleMessage.xwt#/"/>
+    </sections>
+    <sections xmi:id="_70QJkqDJEeSZxfCXzZz3-l" name="SingleLifeline1" sectionFile="ui/SingleLifeline.xwt">
+      <widget href="ui/SingleLifeline.xwt#/"/>
+    </sections>
+  </tabs>
+  <views xmi:id="_70idvaDJEeSZxfCXzZz3-w" name="SingleMessage" sections="_70QJkqDJEeSZxfCXzZz3-w" automaticContext="true">
+    <constraints xsi:type="constraints:SimpleConstraint" xmi:id="_70idvqDJEeSZxfCXzZz3-w" name="isSingleMessage">
+      <constraintType href="ppe:/environment/org.eclipse.papyrus.uml.properties/Model/Environment.xmi#//@constraintTypes.0"/>
+      <properties xsi:type="constraints:ValueProperty" xmi:id="_70idv6DJEeSZxfCXzZz3-w" name="umlClassName" value="Message"/>
+    </constraints>
+  </views>
+  <views xmi:id="_70idwKDJEeSZxfCXzZz3-w" elementMultiplicity="-1" name="MultipleMessage" sections="_70QJk6DJEeSZxfCXzZz3-w" automaticContext="true">
+    <constraints xsi:type="constraints:SimpleConstraint" xmi:id="_70idwaDJEeSZxfCXzZz3-w" name="isMultipleMessage">
+      <constraintType href="ppe:/environment/org.eclipse.papyrus.uml.properties/Model/Environment.xmi#//@constraintTypes.0"/>
+      <properties xsi:type="constraints:ValueProperty" xmi:id="_70idwqDJEeSZxfCXzZz3-w" name="umlClassName" value="Message"/>
+    </constraints>
+  </views>
+  <views xmi:id="_70idvaDJEeSZxfCXzZz3-l" name="SingleLifeline" sections="_70QJkqDJEeSZxfCXzZz3-l" automaticContext="true">
+    <constraints xsi:type="constraints:SimpleConstraint" xmi:id="_70idvqDJEeSZxfCXzZz3-l" name="isSingleLifeline">
+      <constraintType href="ppe:/environment/org.eclipse.papyrus.uml.properties/Model/Environment.xmi#//@constraintTypes.0"/>
+      <properties xsi:type="constraints:ValueProperty" xmi:id="_70idv6DJEeSZxfCXzZz3-l" name="umlClassName" value="Lifeline"/>
+    </constraints>
+  </views>
+  <dataContexts xmi:id="_700xgaDJEeSZxfCXzZz3-w" name="LWSequenceDiagram" label="Sequence Diagram">
+    <elements xmi:id="_700zkKDJEeSZxfCXzZz3-w" name="Message">
```

Comment:
Should this inherit `NamedElement` from the UML properties?  Likewise the `Lifeline`.

Reply:
I didn't do that to avoid pulling in unnecessary features, such as qualified name and visibility. That doesn't mean we couldn't still do that and just not show the features in the xwts, however, I thought it would be more straightforward this way. Let me know what you prefer.

Reply:
Okay, I thought maybe there wouldn't be so many features.  I'm okay with this, then.